backend/capellacollab/models/routes.py

```diff
@@ -0,0 +1,116 @@
+# Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard library:
+import typing as t
+
+# 3rd party:
+from fastapi import APIRouter, Depends
+from requests import Session
+
+# local:
+from . import crud
+from .models import EmptyModel, NewModel, ResponseModel, ToolDetails
+
+# 1st party:
+from capellacollab.core.authentication.database import verify_project_role
+from capellacollab.core.authentication.jwt_bearer import JWTBearer
+from capellacollab.core.database import get_db
+from capellacollab.projects import crud as projects_crud
+
+router = APIRouter()
+
+
+@router.get("/{project_slug}/", response_model=t.List[ResponseModel])
+def get_id(
+    project_slug: str, db: Session = Depends(get_db), token=Depends(JWTBearer())
+) -> t.List[ResponseModel]:
+
+    project = projects_crud.get_slug(db, project_slug)
+    verify_project_role(project.name, token, db)
+    return [ResponseModel.from_model(model) for model in crud.get_all(db, project_slug)]
+
+
+@router.get("/{project_slug}/details/", response_model=ResponseModel)
+def get_slug(
+    project_slug: str,
+    slug: str,
+    db: Session = Depends(get_db),
+    token=Depends(JWTBearer()),
+) -> ResponseModel:
+
+    project = projects_crud.get_slug(db, project_slug)
+    verify_project_role(project, token, db)
```

Comment:
By default `allowed_roles` contains all roles: "user", "manager", "administrator".  In this case, there is no reason for verification because it would be always true. We should define e.g. `allowed_roles=["admin", "manager"]`

Reply:
I removed those useless checks.